compliance_checker/acdd.py

```diff
@@ -6,7 +6,7 @@
 '''
 
 import numpy as np
-from netCDF4 import num2date
+from cftime import num2pydate
```

Comment:
We don't list an explicit dependency on cftime. If we are going to rely on its API we should probably list  `cftime >=1.1.0` in the dependencies.